scripts/test_auto_buy.py

```
@@ -0,0 +1,48 @@
+    try:
+        # 스케줄러의 _execute_auto_buy 함수 직접 호출
+        print("매수 로직 실행 중...")
+        await stock_scheduler._execute_auto_buy(send_slack_notification=False)
+        print()
+        print("=" * 80)
+        print("✅ 매수 테스트 완료")
+        print("=" * 80)
+        print()
+        print("📋 로그 파일 확인: stock_scheduler.log")
+        print()
+        return 0
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)